Packs/RecordedFuture/Integrations/RecordedFutureLists/README.md

```diff
@@ -81,3 +84,63 @@ Add entities to a list, separate entities by commas. "NOTE:" if entity type is s
 | RecordedFuture.List.Entities.id | String | Unique id of the entity in Recorded Future | 
 | RecordedFuture.List.Entities.input_value | String | The value inputted to the command | 
 | RecordedFuture.List.Entities.action_result | String | Entity specific result for the action | 
+
+### recordedfuture-lists-remove-entities
+
+***
+Remove entities from a list, separate entities by commas. "NOTE:" if entity type is specified, only one entity type can be added with each action.
+When adding using IDs use the following for Recorded Future light entities:
++ IPaddress: "ip:x.x.x.x"
++ Domain: "idn:example.xyz"
++ Hash: "hash:examplehashvalue"
++ Email: "email:example@example.xyz"
++ Url: "url:https://example.xyz"
+
+#### Base Command
+
+`recordedfuture-lists-remove-entities`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| list_id | Id of the list that should be removed, can be found by running !recordedfuture-lists-search with the corresponding filters or in the Recorded Future portal. | Required | 
+| entity_ids | Specific ids from Recorded Future separated by comma, For urls containing commas: replace comma with %2C. | Optional | 
+| freetext_names | Freetext names will be matched to Recorded Future ids separated by comma, this alernative will remove the best match in the Recorded Future data. For urls containing commas: escape with %2C. | Optional | 
```

Comment:
```suggestion
| freetext_names | A comma-separated list of freetext names to be matched to Recorded Future IDs. This will remove the best match in the Recorded Future data. For URLs containing commas, escape with %2C. | Optional | 
```